components/IAM/base/external-secrets/konflux-ldap-sa.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: konflux-ldap-sa
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        conversionStrategy: Default
+        decodingStrategy: None
+        key: staging/group-sync/konflux-ldap-sa
+  refreshInterval: 1h
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: appsre-stonesoup-vault
```

Comment:
this ES is trying to extract the secret from a ClusterSecretStore named  "appsre-stonesoup-vault", you installed ESO but did not create any ClusterSecretStore. We will need to create a ClusterSecretStore named appsre-stonesoup-vault and make sure to have the creds to connect to vault

Reply:
Done.